src/CoreFx.Private.TestUtilities/ref/CoreFx.Private.TestUtilities.cs

```diff
@@ -95,6 +95,7 @@ public static partial class PlatformDetection
         public static bool IsUbuntu1710OrHigher { get { throw null; } }
         public static bool IsUbuntu1804 { get { throw null; } }
         public static bool IsWindows { get { throw null; } }
+        public static bool IsWindows10Version1507OrGreater { get { throw null; } } // >= First release RTM
```

Comment:
This should not be needed. The WinRT test in this PR should work on all Windows flavors that support WinRT.

Reply:
previously i get some error on some Win releases...because Win10 is a requirement i think https://docs.microsoft.com/en-us/uwp/api/windows.foundation.collections.stringmap

Reply:
fail on some win flavors `System.TypeLoadException : Could not find Windows Runtime type 'Windows.Foundation.Collections.StringMap'.`

Reply:
I suggest shipping the test on those flavors. PlatformDetection class may help

Reply:
> I suggest shipping the test on those flavors. PlatformDetection class may help

@jkotas  said that this test should work on all Win plat support WinRT...as condition i've added to [test](https://github.com/dotnet/corefx/pull/30809/files#diff-6a2ea6ae5bb24519b398b1a776a30396R535). It's strange...seems that this type is not a first UWP release type.

Reply:
It is not present in Windows 8: https://docs.microsoft.com/en-us/uwp/api/Windows.Foundation.Collections.StringMap
I don't know exactly what's available on Windows Nano but since it's heavily trimmed it doesn't surprise me that it's not there either.

This should work:
`[ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsWinRTSupported), nameof(PlatformDetection.IsNotWindows8x), nameof(PlatformDetection.IsNotWindowsNanoServer)]`

Reply:
Possibly IsWinRTSupported should be false for Nano, but I don't know.

Reply:
> [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsWinRTSupported), nameof(PlatformDetection.IsNotWindows8x), nameof(PlatformDetection.IsNotWindowsNanoServer)]

I agree, it was one of my idea to solve...but i wanted to be sure that the issue was missing type from flavours and not my other errors. I checked the guide https://docs.microsoft.com/en-us/uwp/api/Windows.Foundation.Collections.StringMap (before propose IsWindows10Version1507OrGreater implementation) and it is not "super clear" however...it would have been better list "supported" SO, "Windows 10 requirements" could no exlude other SO...seems a note for "Windows 10" only, but maybe is only my wrong interpretation.
I'll add this conditions thank's a lot for support!